postgis/templates/deployment.yaml

```
@@ -76,7 +78,7 @@ spec:
             - name: DB_USER
               value: {{ .Values.postgres_user | quote }}
             - name: DB_PASS
-              value: {{ .Values.postgres_password | quote }}
+              value: {{ .Values.postgres_password | quote }}             
```

```suggestion
              value: {{ .Values.postgres_password | quote }}
```
Remove white space at the end of the line.

---

Done

---

Also the geoserver and posgist chart version were updated to 0.0.2 in commit [967d82b](https://github.com/Aquaveo/misc-helm-charts/pull/7/commits/967d82bfbfe4b6f2429e0761f49603573e24fe9a)